src/index.js

```diff
@@ -0,0 +1,197 @@
+'use strict';
+
+const state = {
+  temperature: 79,
+  city: 'London',
+  isF: true,
+};
+
+const convertKtoC = (temp) => temp - 273.15;
+const convertKtoF = (temp) => (temp - 273.15) * (9 / 5) + 32;
+const convertCtoF = (temp) => temp * (9 / 5) + 32;
+const convertFtoC = (temp) => (temp - 32) * (5 / 9);
+
+const getLatAndLon = (city) => {
+  return axios
+    .get('http://127.0.0.1:5000/location', {
+      params: {
+        q: city,
+      },
+    })
+    .then((response) => {
+      console.log(response.data);
+      return {
+        lat: response.data[0].lat,
+        lon: response.data[0].lon,
+      };
+    })
+    .catch((error) => {
+      console.log('Error finding the latitude and longitude:', error.response);
+    });
+};
+
+const getWeatherKelvin = (lat, lon) => {
+  return axios
+    .get('http://127.0.0.1:5000/weather', {
+      params: {
+        lat: lat,
+        lon: lon,
+      },
+    })
+    .then((response) => {
+      console.log(response.data);
+      return response.data.current.temp;
+    })
+    .catch((error) => {
+      console.log('Error finding the latitude and longitude:', error.response);
+    });
+};
+
+const handleCityEntered = (event) => {
+  console.log(`City entered ${event}`);
+  const inputCityName = event.target.value;
+  const displayCityName = document.getElementById('cityDisplay');
+  state.city = inputCityName;
+  displayCityName.textContent = state.city;
+};
+
+const increaseTemp = (event) => {
+  state.temperature += 1;
+  colorChange();
+};
+
+const decreaseTemp = (event) => {
+  state.temperature -= 1;
+  colorChange();
+};
+
+const reset = (event) => {
+  // state.temperature = 79; //change it after API
+  state.city = 'London';
+  document.getElementById('city').value = state.city;
+  document.getElementById('cityDisplay').textContent = state.city;
+
+  go(event);
+};
+
+const go = (event) => {
```

Comment:
This method is doing a lot since it gets lat/lon and weather for city, but the method name does not indicate what it's actually doing. Rename `go` to something more descriptive so others know exactly what the method does, maybe something like `getRealCityTemp`